tests/test_virtual_chassis.py

```diff
@@ -24,3 +26,96 @@ def test_connectivity(self, vct):
                 _, out = dv.runcmd(['sh', "-c", "ping -c 5 -W 0 -q %s" % ip])
                 print(out)
                 assert '5 received' in out
+
+    def test_voq_switch(self, vct):
+        """ Test VOQ switch objects configuration """
+        dvss = vct.dvss
+        for name in dvss.keys():
+            if not name.startswith("supervisor"):
```

Comment:
Do we need to skip fabric switch as well?

Reply:
Yes. These configs are available only in line cards config db

Reply:
Can we check switch_type, instead of name, to skip supervisor and fabric switch?

Reply:
Yes. I'll fix it.

Reply:
Fixed